packages/blocks/src/page-block/default/default-page-block.ts

```diff
@@ -161,9 +161,9 @@ export class DefaultPageBlockComponent
   mouseRoot!: HTMLElement;
 
   @state()
-  frameSelectionRect: DOMRect | null = null;
+  private frameSelectionRect: DOMRect | null = null;
 
-  @state()
+  @property()
```

Comment:
> This decorator `property` should only be used for public fields. As public fields, properties should be considered as primarily settable by element users, either via attribute or the property itself.

This field `viewportState` seems to be misused. I think it's better to keep it as it is for now.

```suggestion
  @state()
```

Reply:
https://github.com/toeverything/blocksuite/blob/956afab69811bad17af70eacb9f24e044442f0d9/packages/blocks/src/page-block/default/selection-manager.ts#L547

The `viewportState` attribute will be used outside the component here

Reply:
Alternatively, we can provide a getter to use `viewportState` property outside the component